### PR TITLE
Add kubectl-apply

### PR DESCRIPTION
**Pull request by RammusXu**
## Example

```yaml
jobs:
  deploy:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - name: Kubectl apply
      uses: swaglive/actions/kubectl-apply@master
      with:
        kube_config: ${{ secrets.KUBE_CONFIG }}
        context: gke_asia-east1_edge-tw:ops:ops
        path: templates/bot       # default: .
        kustomize: true           # default: false
        watch: deployment/bot     # optional
        watch_namespace: ops      # default: default
        watch_timeout: 100s       # default: 60s
```

## Features
- Load `KUBE_CONFIG`
- `INPUT_CONTEXT` to choose kubernetes context
- `kustomize` for `apply -k`. Default to `apply -f`
- `watch_*` for `kubectl rollout status`

## Demo
```log
Run swaglive/actions/kubectl-apply@kubectl
  with:
    KUBE_CONFIG: ***
    path: action-demo
    context: gke_swag-2c052_asia-east1_edge-tw:ops:ops
    kustomize: true
    watch: deployment/action-demo
    watch_namespace: ops
    watch_timeout: 60s
/usr/bin/docker run --name d3871e8eb1ee504abc81e8de3f928703c4_a9316f --label 3888d3 --workdir /github/workspace --rm -e INPUT_KUBE_CONFIG -e INPUT_PATH -e INPUT_CONTEXT -e INPUT_KUSTOMIZE -e INPUT_WATCH -e INPUT_WATCH_NAMESPACE -e INPUT_WATCH_TIMEOUT -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/devops-playground/devops-playground":"/github/workspace" 3888d3:871e8eb1ee504abc81e8de3f928703c4
configmap/action-demo unchanged
service/action-demo unchanged
deployment.apps/action-demo configured
Waiting for deployment "action-demo" rollout to finish: 1 old replicas are pending termination...
Waiting for deployment "action-demo" rollout to finish: 1 old replicas are pending termination...
Waiting for deployment "action-demo" rollout to finish: 1 old replicas are pending termination...
deployment "action-demo" successfully rolled out
```

### CODE REVIEW

File: kubectl/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# 2019-10-24
+# GitVersion:"v1.13.11-dispatcher", GitCommit:"2e298c7e992f83f47af60cf4830b11c7370f6668"
+FROM gcr.io/cloud-builders/kubectl@sha256:1023dbcb23ff0776891bd9ed3e3d1e83d35f4c274d8dc2f2017490bfcba38da3
```
Comment (allanlei) [inline kubectl/Dockerfile]: Use a different kubectl.  too fat

**Review by allanlei — CHANGES_REQUESTED** [review]

File: kubectl/Dockerfile (hunk 1 above)
Comment (RammusXu) [reply kubectl/Dockerfile]: Change it to `swaglive/kubectl:1.17`

**Comment by RammusXu** [comment]
Resolve #7

**Review by allanlei — CHANGES_REQUESTED** [review]
Needs to be more Unix like.   `Write programs that do one thing and do it well`.

**Review by allanlei — APPROVED** [review]

**Pull request merged**